src/time.cr

```diff
@@ -262,7 +267,8 @@ struct Time
     Time.new(seconds: seconds, nanoseconds: nanoseconds.to_i, kind: kind)
   end
 
-  def -(other : Time)
+  # Returns the amount of time between *other* and `self`.
```

Comment:
My only concern is that this method doesn't make it 100% clear that the time returned can be negative. "The amount of time between" can easily be interpreted as being `abs(difference)`.

Reply:
Maybe just saying "The amount can be negative if `self` is a Time that happens before *other*`

Reply:
ok